src/monitoring/monitoring.service.ts

```diff
@@ -16,12 +16,20 @@ let versionCache: null | {
   preRelease?: string;
   commit?: string;
 } = null;
-async function getServerVersionFromPackageJson() {
+async function getServerVersionFromPackageJson(): Promise<{
+  major: number;
+  minor: number;
+  patch: number;
+  preRelease?: string;
+  commit?: string;
```

Comment:
Maybe this should be it's own type instead of typed out two times next to each other?